src/gui/mainwindow.cpp

```diff
@@ -1666,7 +1677,8 @@ void MainWindow::showNotificationBaloon(const QString &title, const QString &msg
     QVariantMap hints;
     hints["desktop-entry"] = "qBittorrent";
     QDBusPendingReply<uint> reply = notifications.Notify("qBittorrent", 0, "qbittorrent", title,
-                                                         msg, QStringList(), hints, -1);
```

Comment:
After reading https://specifications.freedesktop.org/notification-spec/latest/ar01s09.html
```
If -1, the notification's expiration time is dependent on the notification server's settings, and may vary for the type of notification. If 0, never expire. 
```
I felt the `-1` is a great choice for many, why would one want to customize it in application instead of at the notification server?